CREDITS.md

```diff
@@ -160,14 +174,17 @@ Contributors to the Firecracker repository:
 - Mihai Stan <stanmihai17cs@gmail.com>
 - milahu <milahu@gmail.com>
 - moricho <ikeda.morito@gmail.com>
+- Muskaan Singla <ec2-user@ip-172-31-1-28.us-west-2.compute.internal>
```

Comment:
Same, can we fix by adding `.mailmap` entry?

Reply:
fixed